index.d.ts

```diff
@@ -276,6 +276,14 @@ declare namespace fastifyGQL {
     preparedOnly: (persistedQueries: object) => PeristedQueryProvider;
     automatic: (maxSize?: number) => PeristedQueryProvider;
   };
+
+  /**
+   * Default error formatter.
+   */
+  const defaultErrorFormatter: (
+    execution: ExecutionResult,
+    context: any
+  ) => { statusCode: number, response: ExecutionResult };
```

Comment:
We would need a test for this type as well. https://github.com/mcollina/fastify-gql/blob/master/test/types/index.ts